libs/spark/src/Unstable_IconButton/Unstable_IconButton.tsx

```diff
@@ -0,0 +1,170 @@
+        boxShadow: `0px 0px 2px 4px ${theme.unstable_palette.teal[300]}`,
+      },
+      ...(props.variant === 'primary' && {
+        backgroundColor: theme.unstable_palette.brand.blue,
+        '&:hover': {
+          backgroundColor: lighten(theme.unstable_palette.brand.blue, 0.1),
+        },
+        '&:active': {
+          backgroundColor: darken(theme.unstable_palette.brand.blue, 0.2),
+        },
+        '&[aria-expanded="true"]': {
+          backgroundColor: theme.unstable_palette.neutral[600],
+        },
+        '&:disabled': {
+          backgroundColor: theme.unstable_palette.neutral[80],
+        },
+      }),
+      ...(props.variant === 'stroked' && {
+        backgroundColor: theme.unstable_palette.neutral[0],
+        borderColor: theme.unstable_palette.neutral[90],
+        '&:hover': {
+          backgroundColor: theme.unstable_palette.neutral[70],
+        },
+        '&:active': {
+          backgroundColor: theme.unstable_palette.blue[100],
+        },
+        '&[aria-expanded="true"]': {
+          backgroundColor: theme.unstable_palette.neutral[600],
+        },
+        '&:disabled': {
+          backgroundColor: theme.unstable_palette.neutral[80],
+          color: theme.unstable_palette.neutral[100],
+        },
+      }),
+      ...(props.variant === 'ghost' && {
+        backgroundColor: 'transparent',
+        '&:hover': {
+          backgroundColor: theme.unstable_palette.neutral[70],
+        },
+        '&:active': {
+          backgroundColor: theme.unstable_palette.blue[100],
+        },
+        '&[aria-expanded="true"]': {
+          backgroundColor: alpha(theme.unstable_palette.neutral[600], 0.8),
+        },
+        '&:disabled': {
+          backgroundColor: alpha(theme.unstable_palette.neutral[200], 0.2),
+        },
+      }),
+
+      ...(props.size === 'small' && {
+        padding: 4,
+      }),
+      ...(props.size === 'medium' && {
+        padding: 12,
+      }),
+    }),
+
+    label: (props: Unstable_IconButtonProps) => ({
+      fontSize: '1.5rem',
+      ...(props.variant === 'primary' && {
+        color: theme.unstable_palette.neutral[0],
+      }),
+      ...(props.variant === 'stroked' && {
+        color: theme.unstable_palette.brand.blue,
+      }),
+      ...(props.variant === 'ghost' && {
+        color: theme.unstable_palette.brand.blue,
+      }),
+      '[aria-expanded="true"] > &': {
+        color: theme.unstable_palette.neutral[0],
+      },
+      ...(props.disabled && {
+        color: theme.unstable_palette.neutral[100],
+      }),
+    }),
+  }),
+  { name: 'MuiSparkUnstable_IconButton' }
+);
+
+const Unstable_IconButton: OverridableComponent<Unstable_IconButtonTypeMap> = React.forwardRef(
+  function Unstable_IconButton(props, ref) {
+    const {
+      classes: classesProp,
+      disabled,
+      size = 'medium',
+      variant = 'primary',
+      // `color` happens to be an html attribute as well; conflicts with Mui prop type
+      color: _color,
```

Comment:
You extract it here but don't use it. And it's not included in the type for possible values. can you explain this?

Reply:
I can follow the typing that variable receives, and its a non-standard attribute off of DOM typings, sepcificaly HTMLAttributes. The props inherit this off of HTMLButton (or whatever it's called) I assume.
![image](https://user-images.githubusercontent.com/25781782/170137125-f0daf782-3fc8-440b-8ec5-bd3aa6d13bdb.png)